compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -469,6 +469,71 @@ class SymbolReferenceTable
    TR::SymbolReference * createTemporary(TR::ResolvedMethodSymbol * owningMethodSymbol, TR::DataType, bool isInternalPointer = false, size_t size = 0);
    TR::SymbolReference * findStaticSymbol(TR_ResolvedMethod * owningMethod, int32_t cpIndex, TR::DataType);
 
+   /** \brief
+    *     Returns a symbol reference for an entity in the source program.
+    *
+    *     Symrefs returned by this function correspond to entities that
+    *     appear the source program. When a symref is created, it is cached
+    *     so that subsequent invocations will return the cached symref
+    *     instead of creating a new one for the same entity. Once created,
+    *     a symref can be returned by both findOrCreateShadowSymbol
+    *     and findOrFabricateShadowSymbol.
+    * 
+    *  \param owningMethodSymbol
+    *     The method that owns the field for which a symbol reference needs to be created.
+    *  \param cpIndex
+    *     Constant pool index.
+    *  \param isStore
+    *     Specifies whether the shadow is generated from a store.
+    *  \return
+    *     Returns a symbol reference created for the field.
+    */
+   TR::SymbolReference * findOrCreateShadowSymbol(TR::ResolvedMethodSymbol * owningMethodSymbol, int32_t cpIndex, bool isStore);
+   /** \brief
```

Comment:
Would you mind adding a space above this comment to separate it further from the declaration above?